src/node/internal/internal_net.ts

```diff
@@ -269,12 +269,18 @@ export function Socket(this: Socket, options?: SocketOptions): Socket {
     throw new ERR_OPTION_NOT_IMPLEMENTED('options.fd');
   }
 
-  if (options.noDelay) {
-    throw new ERR_OPTION_NOT_IMPLEMENTED('truthy options.noDelay');
-  }
-  if (options.keepAlive) {
-    throw new ERR_OPTION_NOT_IMPLEMENTED('truthy options.keepAlive');
-  }
+  // TODO(soon): Support both of these options.
+  //
```

Comment:
I don't think we really have any intention of supporting these. I would just replace this whole block with a comment like:

```
// We do not support the noDelay and keepAlive options at this
// time and will just ignore them if they are passed.
```